### DISCUSSION TITLE
GitHub Suspicious Login Email

### DISCUSSION THREAD

**Discussion by Storm99999** [community/community#102051] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/87811650/d8ee0aca-c0c0-4a85-b961-e8fbab1ffb4d)
Was signed out of github while browsing. Checked my email, told me to reset password. I did. Opened Security Log and was greeted with this.

Is it a petty attempt at stealing my account or is it github's fault?

**Comment by IceyFL** [comment]
I would either change your github to a different email and change the password or change the password to your email and your github
i would assume thats someone trying to steal your account

**Reply by Storm99999** [reply]
github-staff is a staff acount though? I'm not sure

**Comment by Storm99999** [comment]
Note: There were no login attempts made, except mine, but i was signed out and github-staff is a staff account. I have no idea.

**Comment by queenofcorgis** [comment]
Hi there - we responded on your other discussion, but we'll be closing this as it is a duplicate. Thanks for being a part of the community, in the future, please only create one post per topic. 

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
